pkg/scheduler/metrics/metrics.go

```diff
@@ -25,23 +25,26 @@ import (
 
 const schedulerSubsystem = "scheduler"
 
+const BindingType = "binding"
+const SchedulingAlgorithmType ="scheduling_algorithm"
+const E2eSchedulingType = "e2e_scheduling"
+
+type resettableCollector interface {
```

Comment:
I don't think this should be called a `resettableCollector` (because it's just a collector, and not all of them are resettable). Can't you directly just use a prometheus.Collector?

Reply:
done